galaxy.yml

```diff
@@ -16,21 +16,23 @@ readme: README.md
 # A list of the collection's content authors. Can be just the name or in the format 'Full Name <email> (url)
 # @nicks:irc/im.site#channel'
 authors:
-- Dynatrace <support@dynatrace.com>
+  - Dynatrace <support@dynatrace.com>
 
 
 ### OPTIONAL but strongly recommended
 # A short summary description of the collection
-description: This Event source plugin from Dynatrace captures all problems from your Dynatrace tenant and in conjunction with Ansible EDA rulebooks helps to enable auto-remediation in your environment.
+description: |
```

Comment:
Do we use the same galaxy config for both plugins? If so, we need to adjust the description.